grafana/benchmark-runner-ci-status.json

```diff
@@ -27,7 +27,7 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
-  "id": 1,
+  "id": 2,
```

Comment:
What's this?  Is it a version of some kind?  It's really unfortunate that JSON doesn't allow comments :-(

Reply:
Ignore, that changes in the UI that affect the json file